### PR TITLE
added afearonemoju test

### PR DESCRIPTION
**Pull request by AlexisFearon**

### CODE REVIEW

**Comment by afeld** [comment]
Awesome work. Could use tests.

**Comment by AugustHall** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,4 @@
 _site
 Gemfile.lock
+
+_data/spring_2015/afearonemoji
```
Comment (afeld) [inline .gitignore:4]: Hmm, this might not be needed.

File: _data/spring_2015/afearonemoji.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "emoji": "dancer",
+  "introduction": "Developer at 18F by day, dancer by night."
+}
```
Comment (afeld) [inline _data/spring_2015/afearonemoji.json:4]: This file should be named `AlexisFearon.json`...mind re-submitting, but with your own emoji and introduction?

**Pull request merged**